pyproject.toml

```diff
@@ -150,7 +150,7 @@ select = [
     "RUF026", # default factory kwarg
     "TCH",
     "TRY002", # ban vanilla raise (todo fix NOQAs)
-    "TRY302",
+    "TRY203",
```

Comment:
What's going on here?

Reply:
The useless-try-except rule (in our tryceratops category) has been recoded from TRY302 to
TRY203 (https://github.com/astral-sh/ruff/pull/13502). This ensures Ruff's code is consistent with
the same rule in the [tryceratops](https://github.com/guilatrova/tryceratops) linter.